test/com/google/enterprise/adaptor/ApplicationTest.java

```diff
@@ -251,6 +252,15 @@ public void testBasicHttpsListen() throws Exception {
     thrown.expect(java.io.FileNotFoundException.class);
     conn.getContent();
   }
+  
+  private void initSSLKeystores() {
```

Comment:
is this section simply redundant for non-eclipse users? why don't non-ecloipse users need it?  is there something that can be removed for non-eclipse users? some build.xml lines that are no longer necessary for unit testing?